apps/meet-app/backend/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "wso2"
+name = "sample_app"
```

Comment:
```suggestion
name = "meet_app"
```

Reply:
This is just pushing the boilerplates to the repo. Future PRs will include app-specific changes.